hmm.json

```diff
@@ -80,7 +80,7 @@
       "name": "hxCodec",
       "type": "git",
       "dir": null,
-      "ref": "c0c7f2680cc190c932a549c2e2fdd9b0ba2bd10e",
+      "ref": "2aa654b974507ab51ab1724d2d97e75726fd7d78",
```

Comment:
Probably an oversight but that branch is used by funkin.Vis, not hxCodec. You probably meant to use [61b98a7a353b7f529a8fec84ed9afc919a2dffdd](https://github.com/FunkinCrew/hxCodec/commit/61b98a7a353b7f529a8fec84ed9afc919a2dffdd) as the hash.

Reply:
should be fixed now